components/RichText/styles.module.scss

```diff
@@ -24,13 +24,21 @@
     @include heading-1;
 
     margin: $spacing-6 auto $spacing-3;
+
+    strong {
+        font-weight: 900;
+    }
```

Comment:
I believe we should also support the bold font on the story cards, right?
Also worth to consult with @iamhugofelix on that.